tests/Controller/EnMarche/FormationControllerTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Tests\AppBundle\Controller\EnMarche;
+
+use Liip\FunctionalTestBundle\Test\WebTestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\AppBundle\Controller\ControllerTestTrait;
+
+/**
+ * @group functional
+ * @group formation
```

Comment:
We can remove this `@group`, now we don't use this annotation to split the job. 
We use only the `@group functional` to run the `make tf` target

Reply:
done